docs/can-canjs/can-infrastructure.md

```diff
@@ -38,8 +37,22 @@ me.addEventListener("name", function(){ /* ... */ });
 me.dispatch("name");
 ```
 
-[can-event/batch/batch] adds event batching abilities to the [can-event] event system.
-[can-event/async/async] adds asynchronous batched event dispatching to the [can-event] event system.
+## can-queues
+
+A light weight queue system for scheduling tasks, it runs tasks in one of the following queues:
+
+1. [can-queues.notifyQueue] - Tasks that notify observables "deriving" observables that a source value has changed.
```

Comment:
I know this is copied from the `can-queues` docs, but

> Tasks that notify observables "deriving" observables that a source value has changed.

...doesn't really make sense. How about?

> Tasks that notify "deriving" observables that a source value has changed.

Reply:
Fixed!